### DISCUSSION TITLE
Spider Webb's.

### DISCUSSION THREAD

**Discussion by stav242** [blakeblackshear/frigate#1174] [Ask A Question]
Hi, my front driveway camera has a persistent spider webb near it which I've tried removing reappears each night. The Webb flaps around in front of a parked car and frigate keeps notifying me of car object detection.

What options can I tweak to reduce this ? 
![Driveway-1622777432 433279-hs5arg](https://user-images.githubusercontent.com/6066970/120764880-87988380-c510-11eb-9caf-7a318192f5da.jpg)

**Comment by blakeblackshear** [comment]
https://github.com/blakeblackshear/frigate/discussions/1128#discussioncomment-765747

**Comment by blakeblackshear** [comment]
A comment I posted on Discord recently

> create a zone for the entrance of your driveway, not where your car parks. zones can overlap. require that zone for clips/snapshots/notifications. frigate is going to detect motion on lighting changes and find your parked car again. there isnt a good way to limit that without hurting frigates ability to track objects reliably, but it shouldnt create any clips/snapshots/notifications if configured properly. you will get notified of cars that come and go through that zone.

**Comment by Kate222** [comment]
I have the same problem with spider web. I use anti flea liquid spray the areas surround the camera, it stops spider web for a while.

**Comment by highaltidude** [comment]
Spiders Love IR. Spray with diluted Peppermint oil. Sounds weird but it works.

**Comment by jkmaxwell** [comment]
Having dealt with this problem since 2012 on my cameras I have tried every trick from vaseline (no don't do it) to peppermint to motorized whips. Here's the only solution that has seemed to actually work:

https://www.officesupply.com/cleaning-breakroom/cleaning-janitorial-supplies/brooms-dusters-accessories/brushes-dusters/clean-telescopic-cobweb-duster-blue-white-brush-handle/p570233.html

**Comment by picotrain77** [comment]
We use spider off spray designed specifically for this job. We apply it to the camera (apart from the actual lens) and around the camera say 6” around and this keeps them away for a couple of months at a time

**Reply by Josvds** [reply]
Which spray do you use? What is the brand?

**Comment by arhills** [comment]
When I get a warning of someone on my land from the cameras this time of year and there is no one there, I just go out with my feather duster on a stick, remove the cobwebs and I won't get another false alarm for 2 weeks or so.

**Reply by ElectronicBattle** [reply]
Following some expert advice ( I think it was from NickM-27 or hawkeye217  or perhaps even blakeblackshear himself) I put a load of images through my first "Frigate+" training run a few months ago and the resultant model suffers from drastically fewer false triggers due to spider webs compared with the period prior to the model run.
